Cargo.toml

```diff
@@ -50,7 +50,8 @@ normpath = "0.3.2"
 chrono = "0.4"
 once_cell = "1.15.0"
 crossbeam-channel = "0.5.6"
-clap_complete = {version = "4.0.5", optional = true}
+clap_complete = {version = "4.0", optional = true}
```

Comment:
This looks like a wrongly resolved merge conflict?

Reply:
Indeed, my bad I didn't notice the minor version number